### ISSUE TITLE
【EESS】New submissions for Mon, 15 May 23

### ISSUE THREAD

**Issue by Yukeaaa** [Yukeaaa/arxiv-daily#201]
## Keyword: volume render
There is no result 
## Keyword: volumetric render
There is no result 
## Keyword: remote render
There is no result 
## Keyword: hybrid render
There is no result 
## Keyword: raycast
There is no result 
### Unlocking the Potential of Medical Imaging with ChatGPT's Intelligent  Diagnostics
 - **Authors:** Ayyub Alzahem, Shahid Latif, Wadii Boulila, Anis Koubaa
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2305.07429
 - **Pdf link:** https://arxiv.org/pdf/2305.07429
 - **Abstract**
 Medical imaging is an essential tool for diagnosing various healthcare diseases and conditions. However, analyzing medical images is a complex and time-consuming task that requires expertise and experience. This article aims to design a decision support system to assist healthcare providers and patients in making decisions about diagnosing, treating, and managing health conditions. The proposed architecture contains three stages: 1) data collection and labeling, 2) model training, and 3) diagnosis report generation. The key idea is to train a deep learning model on a medical image dataset to extract four types of information: the type of image scan, the body part, the test image, and the results. This information is then fed into ChatGPT to generate automatic diagnostics. The proposed system has the potential to enhance decision-making, reduce costs, and improve the capabilities of healthcare providers. The efficacy of the proposed system is analyzed by conducting extensive experiments on a large medical image dataset. The experimental outcomes exhibited promising performance for automatic diagnosis through medical images.
### Beware of diffusion models for synthesizing medical images -- A  comparison with GANs in terms of memorizing brain tumor images
 - **Authors:** Muhammad Usman Akbar, Wuhao Wang, Anders Eklund
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2305.07644
 - **Pdf link:** https://arxiv.org/pdf/2305.07644
 - **Abstract**
 Diffusion models were initially developed for text-to-image generation and are now being utilized to generate high quality synthetic images. Preceded by GANs, diffusion models have shown impressive results using various evaluation metrics. However, commonly used metrics such as FID and IS are not suitable for determining whether diffusion models are simply reproducing the training images. Here we train StyleGAN and diffusion models, using BRATS20 and BRATS21 datasets, to synthesize brain tumor images, and measure the correlation between the synthetic images and all training images. Our results show that diffusion models are much more likely to memorize the training images, especially for small datasets. Researchers should be careful when using diffusion models for medical imaging, if the final goal is to share the synthetic images.
## Keyword: medical visualization
There is no result 
## Keyword: interactive volume
There is no result 
## Keyword: rendering
There is no result 
## Keyword: cinematic rendering
There is no result 
## Keyword: volume data
There is no result 
## Keyword: remote visualization
There is no result 
## Keyword: direct volume rendering
There is no result 
## Keyword: mobile device
There is no result 
## Keyword: transfer function
There is no result 
### Multi-level Temporal-channel Speaker Retrieval for Robust Zero-shot  Voice Conversion
 - **Authors:** Zhichao Wang, Liumeng Xue, Qiuqiang Kong, Lei Xie, Yuanzhe Chen, Qiao Tian, Yuping Wang
 - **Subjects:** Audio and Speech Processing (eess.AS); Sound (cs.SD)
 - **Arxiv link:** https://arxiv.org/abs/2305.07204
 - **Pdf link:** https://arxiv.org/pdf/2305.07204
 - **Abstract**
 Zero-shot voice conversion (VC) converts source speech into the voice of any desired speaker using only one utterance of the speaker without requiring additional model updates. Typical methods use a speaker representation from a pre-trained speaker verification (SV) model or learn speaker representation during VC training to achieve zero-shot VC. However, existing speaker modeling methods overlook the variation of speaker information richness in temporal and frequency channel dimensions of speech. This insufficient speaker modeling hampers the ability of the VC model to accurately represent unseen speakers who are not in the training dataset. In this study, we present a robust zero-shot VC model with multi-level temporal-channel retrieval, referred to as MTCR-VC. Specifically, to flexibly adapt to the dynamic-variant speaker characteristic in the temporal and channel axis of the speech, we propose a novel fine-grained speaker modeling method, called temporal-channel retrieval (TCR), to find out when and where speaker information appears in speech. It retrieves variable-length speaker representation from both temporal and channel dimensions under the guidance of a pre-trained SV model. Besides, inspired by the hierarchical process of human speech production, the MTCR speaker module stacks several TCR blocks to extract speaker representations from multi-granularity levels. Furthermore, to achieve better speech disentanglement and reconstruction, we introduce a cycle-based training strategy to simulate zero-shot inference recurrently. We adopt perpetual constraints on three aspects, including content, style, and speaker, to drive this process. Experiments demonstrate that MTCR-VC is superior to the previous zero-shot VC methods in modeling speaker timbre while maintaining good speech naturalness.
## Keyword: video retrieval
There is no result 
### Multi-State Inter-Satellite Channel Models
 - **Authors:** Eray Güven, Olfa Ben Yahia, Güneş Karabulut Kurt
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2305.07207
 - **Pdf link:** https://arxiv.org/pdf/2305.07207
 - **Abstract**
 The advancement in satellite networks (SatNets) offers vast opportunities for large-scale connectivity and flexibility. The intersection of inter-satellite communication (ISC) and the developments in rate-hungry enhanced mobile broadband (eMBB) services have resulted in potential ultra-dense deployments such as mega-constellations. The integration of inter-satellite links (ISL) with high spectrum bands requires a well-defined channel model to anticipate possible challenges and provide solutions. In this research, we address the key concerns for defining SatNets channels and establish a comprehensive node-to-node ISL channel model for the Markov state model. The model takes into consideration satellite mobility, celestial body radiation, and cumulative thermal noise. Using this model, we analyze the bit error rate, outage, and achievable capacity of various ISC cases and propose a state-based power allocation method to improve their performance.
### Physical-layer Adversarial Robustness for Deep Learning-based Semantic  Communications
 - **Authors:** Guoshun Nan, Zhichun Li, Jinli Zhai, Qimei Cui, Gong Chen, Xin Du, Xuefei Zhang, Xiaofeng Tao, Zhu Han, Tony Q.S. Quek
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2305.07220
 - **Pdf link:** https://arxiv.org/pdf/2305.07220
 - **Abstract**
 End-to-end semantic communications (ESC) rely on deep neural networks (DNN) to boost communication efficiency by only transmitting the semantics of data, showing great potential for high-demand mobile applications. We argue that central to the success of ESC is the robust interpretation of conveyed semantics at the receiver side, especially for security-critical applications such as automatic driving and smart healthcare. However, robustifying semantic interpretation is challenging as ESC is extremely vulnerable to physical-layer adversarial attacks due to the openness of wireless channels and the fragileness of neural models. Toward ESC robustness in practice, we ask the following two questions: Q1: For attacks, is it possible to generate semantic-oriented physical-layer adversarial attacks that are imperceptible, input-agnostic and controllable? Q2: Can we develop a defense strategy against such semantic distortions and previously proposed adversaries? To this end, we first present MobileSC, a novel semantic communication framework that considers the computation and memory efficiency in wireless environments. Equipped with this framework, we propose SemAdv, a physical-layer adversarial perturbation generator that aims to craft semantic adversaries over the air with the abovementioned criteria, thus answering the Q1. To better characterize the realworld effects for robust training and evaluation, we further introduce a novel adversarial training method SemMixed to harden the ESC against SemAdv attacks and existing strong threats, thus answering the Q2. Extensive experiments on three public benchmarks verify the effectiveness of our proposed methods against various physical adversarial attacks. We also show some interesting findings, e.g., our MobileSC can even be more robust than classical block-wise communication systems in the low SNR regime.
## Keyword: smartphone
There is no result 
## Keyword: medical volume data
There is no result 
## Keyword: webgpu
There is no result 
## Keyword: webgl
There is no result 
## Keyword: pre-rendering
There is no result 
## Keyword: prerendering
There is no result 
## Keyword: motion prediction
There is no result 
## Keyword: incremental learning
There is no result 
## Keyword: svm incremental
There is no result 
## Keyword: nerf
There is no result 
## Keyword: multiorgan
There is no result 
## Keyword: multi-organ
There is no result 
## Keyword: multi organ
There is no result 
## Keyword: segmentation
There is no result 
## Keyword: medical image segmentation
There is no result 
## Keyword: unet
There is no result 
## Keyword: u-net
There is no result 
## Keyword: interactive segmentation
There is no result 
### Doppler-Resilient Design of CAZAC Sequences for mmWave/THz Sensing  Applications
 - **Authors:** Fan Zhang, Tianqi Mao, Zhaocheng Wang
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2305.07234
 - **Pdf link:** https://arxiv.org/pdf/2305.07234
 - **Abstract**
 Ultra-high-resolution target sensing has emerged as a key enabler for various cutting-edge applications, which can be realized by utilizing the millimeter wave/terahertz frequencies. However, the extremely high operating frequency inevitably leads to significant Doppler shift effects, especially for high-mobility applications, causing the degradation of sensing performance with high false alarm rate. To this end, this paper proposes a parameter design methodology of the well-known constant amplitude zero auto correlation (CAZAC) sequences, which aims at enhancing their resilience to Doppler shifts. Specifically, we suppress the sidelobes incurred by Doppler shifts for the peak-to-sidelobe ratio (PSLR) improvement within the range of interest (RoI) of the radar range profile. The Zadoff-Chu (ZC) sequence, as a representative member in the CAZAC family, is firstly considered. The impacts of its root index on range sidelobes are investigated based on number theory. For an arbitrary-length ZC sequence, a feasible range of the root index is derived to satisfy the requirement of PSLR within the scope of RoI. Furthermore, these design guidelines are extended to a general form of CAZAC sequences, where a low-complexity heuristic algorithm is developed for PSLR improvement. Simulation results demonstrate that under severe Doppler shifts, our proposed methodology could enhance the sensing performance by lowering the false alarm rate while maintaining the same detection rate, compared with its classical counterpart.
### A Bayesian Approach to Characterize Unknown Interference Power in  Wireless Networks
 - **Authors:** Mahmoud Zaher, Emil Björnson, Marina Petrova
 - **Subjects:** Signal Processing (eess.SP); Information Theory (cs.IT)
 - **Arxiv link:** https://arxiv.org/abs/2305.07344
 - **Pdf link:** https://arxiv.org/pdf/2305.07344
 - **Abstract**
 The existence of unknown interference is a prevalent problem in wireless communication networks. Especially in multi-user multiple-input multiple-output (MIMO) networks, where a large number of user equipments are served on the same time-frequency resources, the outage performance may be dominated by the unknown interference arising from scheduling variations in neighboring cells. In this letter, we propose a Bayesian method for modeling the unknown interference power in the uplink of a cellular network. Numerical results show that our method accurately models the distribution of the unknown interference power and can be effectively used for rate adaptation with guaranteed target outage performance.
### Unlocking the Potential of Medical Imaging with ChatGPT's Intelligent  Diagnostics
 - **Authors:** Ayyub Alzahem, Shahid Latif, Wadii Boulila, Anis Koubaa
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2305.07429
 - **Pdf link:** https://arxiv.org/pdf/2305.07429
 - **Abstract**
 Medical imaging is an essential tool for diagnosing various healthcare diseases and conditions. However, analyzing medical images is a complex and time-consuming task that requires expertise and experience. This article aims to design a decision support system to assist healthcare providers and patients in making decisions about diagnosing, treating, and managing health conditions. The proposed architecture contains three stages: 1) data collection and labeling, 2) model training, and 3) diagnosis report generation. The key idea is to train a deep learning model on a medical image dataset to extract four types of information: the type of image scan, the body part, the test image, and the results. This information is then fed into ChatGPT to generate automatic diagnostics. The proposed system has the potential to enhance decision-making, reduce costs, and improve the capabilities of healthcare providers. The efficacy of the proposed system is analyzed by conducting extensive experiments on a large medical image dataset. The experimental outcomes exhibited promising performance for automatic diagnosis through medical images.
